### ISSUE TITLE
Resonating Strike Issues (Extra attack and too much damage)

### ISSUE THREAD

**Issue by rfairburn** [SolastaMods/SolastaUnfinishedBusiness#1833]
The wording on Resonating Strike (Which looks like it was made to imitate a non SRD cantrip with Green in the name) is as follows:

```
Resonating Strike: Select one creature within melee range of you and a different creature that you can see within 5 feet of main target. You make a melee attack with current weapon against first creature. On a hit, the target suffers the weapon attack's normal effects, and resonating sound leaps to second target, dealing thunder damage equal to your spellcasting ability modifier.
At 5th level, the melee attack deals extra 1d8 thunder damage to the target on a hit, and the thunder damage to the second creature increases to 1d8 + your spellcasting ability modifier. Both damage rolls increase by 1d8 at 11th and 17th levels.
```

So what I would expect at level one with an 18 int and an 18 strength would be that my first target is hit for 1d8+4 slashing with a long sword and the second target takes 4 thunder damage.

What is happening instead is that if both targets are in melee range of me, instead I do 1d8+4 slashing to the first target and get a second attack of 1d8+4 slashing on the second target as well (that shouldn't happen) plus the 4 thunder damage (which is fine).

There is a problem as the spell scales up as well.  Once I reach 5th level, I get the 1d8+4 slashing and 1d8 thunder on the first target as expected.  However, on the second target, I am seeing again that I still get the 1d8+4 slashing attack if I hit, and I am getting 2d8+4 thunder damage instead of 1d8+4 thunder total with no slashing that is expected.

I have not tested the scaling at 11th or 17th levels.

Steps to reproduce:

1. Make a character that has resonating strike
2. Attack a target with a second target also in melee range of you and within one square of the original target
3. View damage on hit
4. Level character to 5 (and then 11,17) via preferred method and repeat steps 1-3 to see the extra scaling

I tried it with 2 different characters (one Warlock, one Inventor) with the same results on both.

**Comment by rfairburn** [comment]
This was tested in 1.4.30.8.  I have not had an opportunity to test today's release yet to see if the issue persists.

**Comment by ThyWoof** [comment]
It’s been fixed here https://github.com/SolastaMods/SolastaUnfinishedBusiness/commit/b7ec5491fe4bbf9b52f4d69e9bc7d33738bce9e2

it’s an off progression with 4/6/5/5. My progression routine only considers same step. Fixed with a hack

**Comment by rfairburn** [comment]
Verified it works at level 5 building latest dev.  Also saw you released 1.4.30.10, will update to that.  Thanks for everything you do!
